metpy/calc/tests/test_calc_tools.py

```diff
@@ -202,7 +202,6 @@ def test_log_interp():
     assert_array_almost_equal(y_interp, y_interp_truth, 7)
 
 
-@pytest.fixture
```

Comment:
Would it be too painful to leave these as fixtures and update their use like you did in `test_thermo.py`?

Reply:
Nevermind, I just read what you said.